lib/profile_component/controls/commands/initiate.rb

```diff
@@ -0,0 +1,20 @@
+module ProfileComponent
+  module Controls
+    module Commands
+      module Initiate
+        def self.example
+          initiate = ProfileComponent::Messages::Commands::Initiate.new
+
+          initiate.profile_id = Profile.id
+          initiate.name = 'some name'
+          initiate.email = 'email@some_domain.com'
+          initiate.avatar_url = 'http://some_domain.com/images/some_avatar.jpg'
+          initiate.github_id = 'some_github_id'
+          initiate.time = Controls::Time::Effective.example
+
+          initiate
+        end
```

Comment:
Use the control values you already defined in `lib/profile_component/controls/commands/profile.rb`